src/Microsoft.Restier.Samples.Northwind/Models/NorthwindApi.cs

```diff
@@ -114,4 +115,114 @@ private class NorthwindModelExtender : IModelBuilder
             }
         }
     }
+
+    [EnableRoleBasedSecurity]
+    [Grant(ApiPermissionType.All, On = "Customers")]
+    [Grant(ApiPermissionType.All, On = "Products")]
+    [Grant(ApiPermissionType.All, On = "CurrentOrders")]
+    [Grant(ApiPermissionType.All, On = "ExpensiveProducts")]
+    [Grant(ApiPermissionType.All, On = "Orders")]
+    [Grant(ApiPermissionType.All, On = "Employees")]
+    [Grant(ApiPermissionType.All, On = "Regions")]
+    [Grant(ApiPermissionType.Inspect, On = "Suppliers")]
+    [Grant(ApiPermissionType.Read, On = "Suppliers")]
+    [Grant(ApiPermissionType.All, On = "ResetDataSource")]
+    public class NorthWindApi2
+    {
+        public static void Configure()
```

Comment:
No virtual method to override like in ApiBase, but a static method seems not good enough either.